### PR TITLE
FIX - deploy images updates SSM

### PR DESCRIPTION
**Pull request by dgraeber**
### Description:

Correct orbit deploy images to update SSM.

### Issue Reference URL

https://github.com/awslabs/aws-orbit-workbench/issues/1255
https://github.com/awslabs/aws-orbit-workbench/issues/1260
https://github.com/awslabs/aws-orbit-workbench/issues/1261
https://github.com/awslabs/aws-orbit-workbench/issues/1262

----------
### Do not change content below. Mark applicable check boxes only.

Thank you for submitting a contribution to AWS Orbit Workbench.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there a ISSUE associated with this PR?
- [ ] Does your PR title start with ISSUE-XXXX where XXXX is the issue number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [ ] Has your PR been rebased against the latest commit within the target branch (typically 'main')?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you ensured that the full suite of tests is executed?
- [ ] Have you written or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under Apache License 2.0?

### CODE REVIEW

File: images/jupyter-user/Dockerfile [hunk 1]
```diff
@@ -144,7 +144,7 @@ RUN conda run pip install -r /opt/orbit/aws-orbit_jupyter-user/aws-orbit/require
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/aws-orbit
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/aws-orbit-sdk
 RUN conda run pip install /opt/orbit/aws-orbit_jupyter-user/jupyterlab_orbit
-RUN conda run pip install softwarelabs-remote-toolkit==0.2.0
+RUN conda run pip install softwarelabs-remote-toolkit~=0.1.0
```
Comment (srinivasreddych) [inline images/jupyter-user/Dockerfile:147]: Noticed that we have pushed `0.1.1`version of `slrt`

File: samples/manifests/plugins/dev-env-with-plugins.yaml [hunk 2]
```diff
@@ -10,16 +10,16 @@ SharedEfsSgId: !SSM ${/orbit-f/dev-env/resources::SharedEfsSgId}
...
-    JupyterUser:
-        Repository: !ENV ${JUPYTER_USER_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/jupyter-user}
-        Version: !ENV ${VERSION::latest}
-    OrbitController:
-        Repository: !ENV ${ORBIT_CONTROLLER_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/orbit-controller}
-        Version: !ENV ${VERSION::latest}
-    UtilityData:
-        Repository: !ENV ${UTILITY_DATA_REPO::public.ecr.aws/v3o4w1g6/aws-orbit-workbench/utility-data}
-        Version: !ENV ${VERSION::latest}
+#Images:
```
Comment (srinivasreddych) [inline samples/manifests/plugins/dev-env-with-plugins.yaml:13]: Can you remove the commented section

File: images/jupyter-user/Dockerfile (hunk 1 above)
Comment (dgraeber) [reply images/jupyter-user/Dockerfile:147]: Yep...I can update

File: samples/manifests/plugins/dev-env-with-plugins.yaml (hunk 2 above)
Comment (dgraeber) [reply samples/manifests/plugins/dev-env-with-plugins.yaml:13]: I was leaving for reference...but can delete

**Review by srinivasreddych — APPROVED** [review]

**Pull request merged**